lib/src/input/figma/entities/style/fill/figma_fill_factory.dart

```diff
@@ -13,7 +13,9 @@ class FigmaFillFactory {
     for (var fill in _fills) {
       if (fill.type == json['type']) {
         // Fix for opacity for global colors
-        json['color']['a'] = json['opacity'];
+        if (fill.type == 'SOLID') {
```

Comment:
If this is specifically for solid, maybe we should move this logic to the `createFigmaFill` method of `SolidFillType`

Reply:
Sounds good & done 👍